src/tools.rs

```diff
@@ -272,30 +272,26 @@ async fn maybe_warn_on_outdated(context: &Context, now: i64, approx_compile_time
 
 /* Message-ID tools */
 
-/// Generate an ID. The generated ID should be as short and as unique as possible:
-/// - short, because it may also used as part of Message-ID headers or in QR codes
-/// - unique as two IDs generated on two devices should not be the same. However, collisions are not world-wide but only by the few contacts.
+/// Generate an unique ID.
 ///
-/// IDs generated by this function are 66 bit wide and are returned as 11 base64 characters.
+/// The generated ID should be short but unique:
+/// - short, because it used in Message-ID and Chat-Group-ID headers and in QR codes
+/// - unique as two IDs generated on two devices should not be the same. However, collisions are not world-wide but only by the few contacts.
```

Comment:
The second sentence is a bit hard to understand. Probably it's meant that we don't protect from collisions globally, but only at account and chat scale.